tkg/client/validate.go

```diff
@@ -1874,12 +1874,16 @@ func getDockerBridgeNetworkCidr() (string, error) {
 }
 
 // ConfigureAndValidateAviConfiguration validates the configuration inputs of Avi aka. NSX Advanced Load Balancer
-func (c *TkgClient) ConfigureAndValidateAviConfiguration() error {
+func (c *TkgClient) ConfigureAndValidateAviConfiguration(clusterName string) error {
 	aviEnable, _ := c.TKGConfigReaderWriter().Get(constants.ConfigVariableAviEnable)
 	// ignoring error because AVI_ENABLE is an optional configuration
 	if aviEnable == "" || aviEnable == "false" {
 		return nil
 	}
+
+	if len(clusterName) > constants.AkoMaxAllowedClusterNameLen {
```

Comment:
Do we check if it's clusterclass type cluster here?

Reply:
We don't need to do that, if cluster is not clustercalss based, ako operator sercert can't find, since it is only trying to retrieve akoo v2 addon secret content

Reply:
They have different names